premailer/premailer.py

```diff
@@ -417,6 +417,12 @@ def transform(self, pretty_print=True, **kwargs):
                         parent.attrib[attr].startswith('cid:')
                     ):
                         continue
+                    if parent.attrib[attr].startswith("//"):
+                        try:
+                            protocol = self.base_url.split("//")[0]
+                        except IndexError:
+                            protocol = "http"
```

Comment:
Mainly from the fact I'm using the same django template for both a web view and a mail, that I'm using premailer on both cases (because I want te be able, as far as possible, to detect a possible problem in the premailer transformation of my e-mail before sending), that the web template has some js in it and that the js is linked with //. I was just surprised when my // link did not work, and submitted the issue. If finally it's not an issue, we can reject the PR and call it a day. I'll put https links instead of // (or not premail the template for the web) and I'll be ok.

I don't know if could be useful, but I've seen a situation where a website used both http and https and most static files (including images) (stored on another domain) were linked using // to not mix http (which had a lot of traffic) and https (which didn't). But then I can't say it's a frequent setup, so it may not be worth the effort...